pkg/volume/csi/csi_client.go

```diff
@@ -177,10 +178,30 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 	accessibleTopology map[string]string,
 	err error) {
 	klog.V(4).Info(log("calling NodeGetInfo rpc"))
-	if c.nodeV1ClientCreator != nil {
-		return c.nodeGetInfoV1(ctx)
-	} else if c.nodeV0ClientCreator != nil {
-		return c.nodeGetInfoV0(ctx)
+
+	backoff := gensupport.DefaultBackoffStrategy()
```

Comment:
Let's use the `wait.ExponentialBackoff(...)` from the `k8s.io/apimachinery/pkg/util/wait` library.